### ISSUE TITLE
Display per-record timestamp in navigation history

### ISSUE THREAD

**Issue by bwims** [kiwibrowser/src.next#384]
Hello,

Due to a message on login activity I received from Google, it would be very useful if I could see the timestamp of history records in Kiwi, as you can on PC-based browsers.

Is this possible?

Thanks!

**Comment by KaKi87** [comment]
History records are already grouped by day, isn't it enough ?

**Comment by bwims** [comment]
No, for my purposes it is quite time critical.   Is that your way of saying
that the time is not available?  It must be there somewhere, otherwise PC
browsers wouldn't display it?

**Comment by kiwibrowser** [comment]
Yes I agree it's important and needed for verifying if it was you or not

**Comment by KaKi87** [comment]
Well, if you did log in at least one time that day, with an IP adress located near where you've been that day, then it's you.

If you did not log in at all that day, then it's not you.

Also how would you forget about typing your password ?

I'm not saying that the feature is useless, but I think you can already get a conclusive answer without it right now.

**Comment by kiwibrowser** [comment]
No no, but I understand, it's normal we need the time. It may not be obvious whether it is you or not (and it is quite common to try multiple passwords), and beyond that, he was nice to explain the whole context. Overall, it's well logical feature :)

**Comment by bwims** [comment]
I'm not sure why you are going down the hallowed route of "why would you
want to do it that way" instead of responding to the essence of the
question.
It's pointless to argue whether my particular needs are valid.  I'm sure
with a specific thought experiment, it would be possible to find a
situation where time as well as date is critical.

Let's assume that there is an idiosyncratic case where it is important to
know the time as well as the date.  The point is, can it be determined?  If
the data is there in the history, a possible solution could be to export
the history for analysis via program or other.

Thanks
B.

**Comment by bwims** [comment]
I have a solution.  https://chrome.google.com/webstore/detail/export-chrome-history/dihloblpkeiddiaojbagoecedbfpifdj?hl=en
is the chrome plugin to export history to CSV or JSON.

As kiwi thankfully supports plugins, I was able to dump the history that way.   You certainly do include the time, so it's a mystery to why you don't display it, at least as an option.  I realise Android was designed for teeny weeny screens, so I suppose there may be lunatic Google constraints.

Thanks.

**Comment by tvqt** [comment]
just dropping in to say that this feature would be super neat if it was built-in! 

The extension that @bwims mentions is useful, however, downloading another CSV or JSON file every time one wants to look at the times in the history (in my case, something I do every day) is quite tedious! 
Perhaps there could be a way of viewing the desktop version of the history? I assume it still exists in the source code, it's just a matter of linking to it?

**Comment by NuraliMedeu** [comment]
Good news! There is an extension called [Better History](https://chrome.google.com/webstore/detail/better-history/egehpkpgpgooebopjihjmnpejnjafefi) on Chrome Webstore. Even though this is not a built-in solution, it's more user-friendly than @bwims ' suggestion.

**Comment by davidasunmo** [comment]
@bwims Why did you close this issue even though the requested feature has not been implemented? I also want this enhancement to be done, and I'd prefer not to have to create a duplicate issue just because this one was closed

**Comment by bwims** [comment]
I have reopened this due to a user wanting a response.

**Comment by davidasunmo** [comment]
Thank you very much!
